homework/index.js

```diff
@@ -0,0 +1,64 @@
+function getPromise() {
+  const ret = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+    .then(response => response.json());
+  return ret;
+}
+
+
+function populate(Prom) {
+  const selectContent = document.getElementById('liste').value;
+  Prom.then((result) => {
+    result.forEach((x) => {
+      if (x.name === selectContent) {
+        document.getElementById('show').innerHTML = `
+         <b>Repository :</b>   <a href="">${x.name} </a><br>
+          <b>Descripion :</b>   ${x.description} <br>
+          <b>Forks :</b>   ${x.forks} <br>
+          <b>Updated :</b>   ${new Date(x.updated_at).toLocaleString('en-US')} <br>`;
+      }
+    });
+  });
+}
+
+
+function loadPage(Prom) {
+  Prom.then((result) => {
+    const nameArray = [];
+    // make an array of all the names
+
+    result.forEach((item) => {
+      nameArray.push(item.name);
+    });
+    //  populate the select with the elements in the array
+
+    nameArray.forEach((x) => {
+      const node = document.createElement('OPTION');
```

Comment:
nit-pick: let's use `option` (lower case for elements)

Reply:
Another tip: a `<select>` element has an `add()` method, to which you can pass an `<option>` element directly. And an `<option>` element has a `value` and `text` property which you can set directly. That way you don't have to create a textNode child. More info & example: https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/add